hack/run-e2e.sh

```diff
@@ -37,8 +37,11 @@ REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 export KUBECONFIG=${KARMADA_APISERVER_KUBECONFIG}
 export PULL_BASED_CLUSTERS=${PULL_BASED_CLUSTERS}
 
+POLL_INTERVAl_VALUE=${POLL_INTERVAl_VALUE:-"5"}
+POLL_TIMEOUT_VALUE=${POLL_TIMEOUT_VALUE:-"300"}
+
 set +e
-ginkgo -v --race --trace --fail-fast -p --randomize-all ./test/e2e/
+ginkgo -v --race --trace --fail-fast -p --randomize-all ./test/e2e/ -- --poll-interval-value=${POLL_INTERVAl_VALUE} --poll-timeout-value=${POLL_TIMEOUT_VALUE}
```

Comment:
The two flags may not need to be added for now.

If needed, we can add it later.